lang/csharp/build.sh

```diff
@@ -76,6 +76,35 @@ do
       cp -pr build/doc/* ${ROOT}/build/avro-doc-${VERSION}/api/csharp
       ;;
 
+    release)
+      # "dist" step must be executed before this step
```

Comment:
What do you think about the following:

 extract all the logic in `dist)` to a function, so it will look like:
```bash
...
dist)
   dist()
release)
  release()
...
```

This way the first few lines of `release()` could check if there are artifacts created by `dist()` and call `dist()` if there are no such. Something like:
```bash
function release() {
  if [ ! -f .../some/file ]; then
    dist()
  fi

  # release stuff
}
```